assets/src/admin-import-settings.scss

```diff
@@ -34,30 +35,33 @@ ul {
 }
 
 .log-table {
-    width: 100%;
-    &--large {
-        font-size: 1rem;
-    }
-    thead {
-        position: sticky;
-        top: 2rem;
-        z-index: 30;
-        background: #ddd;    
-    }
-    td, th {
-        text-align: left !important;
-    }
-    .error td {
-        color: #b00;
-    }
-    &__empty {
-        display: none;
-        td {
-            padding: 1rem !important;
-            text-align: center !important;
-            font-family: inherit !important;
-        };
-    }
+	margin: 0 -20px !important;
+	width: calc(100% + 40px) !important;
+	&--large {
+		font-size: 1rem;
+	}
+	thead {
+		position: sticky;
+		top: 2rem;
+		z-index: 30;
+		background: #f0f0f0;
+	}
+	td,
+	th {
+		padding: 10px 20px !important;
+		text-align: left !important;
+	}
+	.error td {
+		color: #b00;
+	}
+	&__empty {
+		display: none;
+		td {
+			padding: 1rem !important;
+			text-align: center !important;
+			font-family: inherit !important;
+		}
+	}
 }
```

Comment:
Something is up with the whitespace config, or lack of config. My editor reads them as spaces. I've used a prettier config to address in other projects, will see if that is appropriate here.